www/src/pages/index.js

```diff
@@ -84,7 +84,7 @@ const ButtonToolbar = styled('div')`
     margin: -1rem;
 
     & > * {
-      width: 100%;
+      width: 90%;
```

Comment:
I think it's better to use `calc()` here because the width does not account for the margins hence the overflow